requirements/edx/github.txt

```diff
@@ -75,7 +75,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/edx/edx-ora2.git@1.1.5#egg=ora2==1.1.5
+git+https://github.com/edx/edx-ora2.git@8d48f40ac14faa391dc56892075f5059ef2e6602#egg=ora2==1.1.5b
```

Comment:
author's note: this is necessary until https://github.com/edx/edx-ora2/pull/912 is merged